packages/discord.js/src/util/Util.js

```diff
@@ -105,8 +105,42 @@ function escapeMarkdown(
       .join(codeBlock ? '\\`\\`\\`' : '```');
   }
   if (!inlineCodeContent) {
-    return text
-      .split(/(?<=^|[^`])`(?=[^`]|$)/g)
+    const noTriple = text.split("```");
```

Comment:
instead of changing it for everyone, u can check the `process.version`  and do it for only bun users.